boards/DCM/Src/main.c

```diff
@@ -615,6 +624,7 @@ void RunTaskCanRx(void const *argument)
         struct CanMsg message;
         Io_SharedCan_DequeueCanRxMessage(&message);
         Io_CanRx_UpdateRxTableWithMessage(can_rx, &message);
+        Io_SharedErrorTable_SetErrorsFromCanMsg(error_table, &message);
```

Comment:
if ur doing this, can you add to the PR description and also close shevchuks PR?

Reply:
done